workspaces/ui-v2/src/optic-components/hooks/diffs/LearnInitialBodiesMachine.ts

```diff
@@ -166,7 +166,7 @@ export function recomputeCommands(ctx: InitialBodiesContext): any[] {
     commands.push(
       AddContribution(
         getEndpointId({ method: ctx.method, pathId: ctx.pathId }),
-        'PURPOSE',
+        'purpose',
```

Comment:
I'm not 100% here, but I believe this should be lowercase (spectacle / rust seems to accept lowercase for me)

Reply:
Yes. It should be. We may want to make a constants file somewhere so we’re not reusing the string directly